ooni/reporter.py

```diff
@@ -319,12 +325,15 @@ def createReport(self):
             # the backend.
             'content': content
         }
+        # import values from the environment
+        request.update([(k.lower(),v) for (k,v) in os.environ.iteritems()
+                        if k.startswith('PROBE_')])
```

Comment:
That's right, it passes deployment-specific environment variables back to the server.